elements/pfe-readtime/README.md

```diff
@@ -0,0 +1,71 @@
+# PatternFly Element | Readtime element
+It takes in the word count of a given section and does a calculation on that number to return an estimated read time.
+
+## Usage
+
+```html
+    <!-- Default slot -->
+    <pfe-readtime for="readtime1">-minute readtime</pfe-readtime>
+```
+
+### Accessibility
+Explain how this component meets accessibility standards.
```

Comment:
This is boilerplate text so it will need to be updated to include details about how this attribute meetings accessibility requirements.